src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND_OF_NUMBERS = 100;
+    private int randomNumberForBall = new Random().nextInt(BOUND_OF_NUMBERS);
+    private ColorSupplier randomColor = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(randomNumberForBall, randomColor.getRandomColor());
+    }
+}
```

Comment:
add empty line, you had two empty lines in the end of the file in the previous version, I asked you to remove only one